packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -593,77 +630,80 @@ export const addAccountMetadata =
 /**
  * Generate device master-key
  * */
-export const setDeviceMetadataKey = () => async (dispatch: Dispatch, getState: GetState) => {
-    if (!getState().metadata.enabled) return;
-    const { device } = getState().suite;
-    if (!device || !device.state || !device.connected) return;
-
-    // mater key already exists
-    if (device.metadata.status === 'enabled') return;
-
-    const result = await TrezorConnect.cipherKeyValue({
-        device: {
-            path: device.path,
-            state: device.state,
-            instance: device.instance,
-        },
-        useEmptyPassphrase: device.useEmptyPassphrase,
-        path: METADATA.ENABLE_LABELING_PATH,
-        key: METADATA.ENABLE_LABELING_KEY,
-        value: METADATA.ENABLE_LABELING_VALUE,
-        encrypt: true,
-        askOnEncrypt: true,
-        askOnDecrypt: true,
-    });
+export const setDeviceMetadataKey =
+    (version: number) => async (dispatch: Dispatch, getState: GetState) => {
```

Comment:
idea: I would maybe consider binding `version` with `TrezorConnect.cipherKeyValue` parameters. I't could be part of config/constants. of course not important, feel free to dismiss.